code/game/objects/items/weapons/RCD.dm

```diff
@@ -65,6 +259,8 @@
 	var/static/list/door_types_ui_list = list()
 	/// An associative list containing all station accesses. Includes their name and access number. For use with the UI.
 	var/static/list/door_accesses_list = list()
+	/// The list of potential RCD actions.
+	var/static/list/possible_actions = subtypesof(/datum/rcd_act)
```

Comment:
```suggestion
	var/static/list/possible_actions = subtypesof(/datum/rcd_act)
```
Set this in init. DM will probably regenerate this every time a new RCD is newd (Internal new sets up lists and stuff)